### DISCUSSION TITLE
请问最新版本何时上传CTAN？

### DISCUSSION THREAD

**Discussion by YDX-2147483647** [hushidong/biblatex-gb7714-2015#212] [General]
今年四月 #193 已经辛勤解决了如下虚假警告。

```
Package biblatex Warning: Macro 'byeditor+others' already defined.
(biblatex)                Using \renewbibmacro.
```

现在都立冬了，该补丁似乎仍未上传到 CTAN？目前 [CTAN: Package biblatex-gb7714-2015](https://www.ctan.org/pkg/biblatex-gb7714-2015) 显示最新版是 1.1r 2024-05-15。

请问这个补丁何时才能上传到 CTAN 呢？

**Comment by hushidong** [comment]
下周吧，之前是等看看《综合性期刊...》那个样式的反馈。

告警不影响使用的。

**Reply by YDX-2147483647** [reply]
好，辛苦了！
